topo_processor/stac/collection_store.py

```diff
@@ -0,0 +1,14 @@
+from .collection import Collection
+
+
+class CollectionStore:
+    collections = {}
```

Comment:
You either need to make these static, or make a single instance of the collection store.

Otherwise everytime you do `CollectionStore()` you get a whole new collection store

Reply:
done making it static